tasks/agent.yml

```diff
@@ -66,6 +66,19 @@
   with_items:
     "{{ bamboo_agent.capabilities }}"
   when: (item.source is defined and item.source == "repository")
+  when: >
+    (item.source is defined and item.source == "repository") and
+    (ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux')
+
+- name: install apt packages
+  yum:
```

Comment:
You may use "apt" module - or we simply switch to "package" module?

Reply:
Yes, this is a mistake. Didn't know the package module, this is the better solution, imho. I'll change it to that one.